src/meta/processors/admin/CreateBackupProcessor.cpp

```diff
@@ -18,11 +18,11 @@ CreateBackupProcessor::spaceNameToId(const std::vector<std::string>* backupSpace
   folly::SharedMutex::ReadHolder rHolder(LockUtils::spaceLock());
   std::unordered_set<GraphSpaceID> spaces;
 
-  if (backupSpaces != nullptr) {
+  bool allSpaces = backupSpaces == nullptr || backupSpaces->empty();
+  if (!allSpaces) {
     DCHECK(!backupSpaces->empty());
```

Comment:
DCHECK can be deleted? because it has checked above

Reply:
OK